orttraining/orttraining/python/ort_trainer.py

```diff
@@ -515,7 +515,7 @@ class ORTTrainer():
     def __init__(self, model, loss_fn, model_desc, training_optimizer_name, map_optimizer_attributes,
                  learning_rate_description, device, gradient_accumulation_steps=1, postprocess_model=None,
                  world_rank=0, world_size=1, use_mixed_precision=False, allreduce_post_accumulation=False,
-                 global_step=0, get_lr_this_step=None, loss_scaler=None, partition_optimizer=False):
+                 global_step=0, get_lr_this_step=None, loss_scaler=None, partition_optimizer=False, opset_version=10):
```

Comment:
I definitely see the need to be able to test different opset versions internally. But do we want to expose this to public API user? I don't think the user will be able to make an informed decision on which opset to use. It will be nice if we can make the choice internally. That said, we need this control for testing and maybe for internal experimentation.So, can this be an internal use only parameter?

Reply:
Resolved by renaming to `_opset_version` as per offline discussion.